Formula/qwt-qt5.rb

```diff
@@ -0,0 +1,103 @@
+class QwtQtAT5 < Formula
```

Comment:
The file also needs to be renamed to `qwt-qt@5.rb` for this

Reply:
I get an error about missing unversioned qwt-qt.rb